codalab_service.py

```diff
@@ -366,6 +366,20 @@ def has_callable_default(self):
     ),
     # Public workers
     CodalabArg(name='public_workers', help='Comma-separated list of worker ids to monitor'),
+    CodalabArg(
+        name='recaptcha_site_key',
+        env_var='REACT_APP_CODALAB_RECAPTCHA_SITE_KEY',
+        type=str,
+        default='6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI',  # test key provided by Google, will always pass
+        help='API site key needed for recaptcha',
+    ),
+    CodalabArg(
+        name='recaptcha_secret_key',
+        env_var='CODALAB_RECAPTCHA_SECRET_KEY',
```

Comment:
We don't need to specify env_var here because it will be set to `CODALAB_RECAPTCHA_SECRET_KEY` by default

Reply:
Suggestions applied in the new commits.